src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -82,25 +82,39 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Attempts to estimate library complexity from sequence of read pairs alone. Does so by sorting all reads " +
-                "by the first N bases (5 by default) of each read and then comparing reads with the first " +
-                "N bases identical to each other for duplicates.  Reads are considered to be duplicates if " +
-                "they match each other with no gaps and an overall mismatch rate less than or equal to " +
-                "MAX_DIFF_RATE (0.03 by default).\n\n" +
-                "Reads of poor quality are filtered out so as to provide a more accurate estimate. The filtering " +
-                "removes reads with any no-calls in the first N bases or with a mean base quality lower than " +
-                "MIN_MEAN_QUALITY across either the first or second read.\n\n" +
-                "Unpaired reads are ignored in this computation.\n\n" +
-                "The algorithm attempts to detect optical duplicates separately from PCR duplicates and excludes " +
-                "these in the calculation of library size. Also, since there is no alignment to screen out technical " +
-                "reads one further filter is applied on the data.  After examining all reads a Histogram is built of " +
-                "[#reads in duplicate set -> #of duplicate sets] all bins that contain exactly one duplicate set are " +
-                "then removed from the Histogram as outliers before library size is estimated.",
-        usageShort = "Estimates library complexity from the sequence of read pairs",
+        usage = EstimateLibraryComplexity.USAGE_SUMMARY + EstimateLibraryComplexity.USAGE_DETAILS,
+        usageShort = EstimateLibraryComplexity.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class EstimateLibraryComplexity extends AbstractOpticalDuplicateFinderCommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Tool for estimating library complexity.   ";
+    static final String USAGE_DETAILS = "This tool is a quality metric for a sequencing library preparation." +
```

Comment:
"This tool is a quality metric" -> "This tool calculates quality metrics"

Metrics are the numbers produced by the tools. The tools themselves are not metrics.